GrowthViz-adults.py

```diff
@@ -74,13 +75,15 @@
 # 
 # The following cell import functions created for the tool to asssist in data analysis. Some of the functions generate charts used in this tool. The chart code may be modified to change the appearance of plots without too much risk of breaking things. Other functions transform DataFrames and changing those will very likely cause things to break. If you are unable to tell the difference in the functions by looking at the code, it is probably best to leave them unmodified.
 
-# In[6]:
+# In[13]:
 
 
+sys.path.insert(0, './tests')
```

Comment:
Should be able to remove this when `check_data` moves.

Reply:
Will we need it back in when we move processdata.py, etc. to a subfolder?

Reply:
Once we have it as something like `growthviz/processdata.py`, we'll just need to import it as `from growthviz import processdata`.